etc/systemd/system-generators/zfs-mount-generator.in

```diff
@@ -107,6 +113,8 @@ process_line() {
   p_systemd_requiredby="${18}"
   p_systemd_nofail="${19}"
   p_systemd_ignore="${20}"
+  # Reset IFS so splitting works normally again.
+  IFS="$OIFS"
```

Comment:
Everything between here and `set -- $1` is protected by quotes, so it should be safe to reset the IFS right after that `set` statement, right?